crowbar_framework/app/models/service_object.rb

```diff
@@ -1492,14 +1492,15 @@ def chef_daemon(action, node_list)
   private
 
   def wait_for_chef_clients(node_name, options = {})
-    options = if options.fetch(:logger)
+    use_logger = options.fetch(:logger)
+    options = if use_logger
       {:logger => @logger}
     else
       {}
     end
-    @logger.debug("wait_for_chef_clients: Waiting for already running chef-clients on #{node_name}.")
+    @logger.debug("wait_for_chef_clients: Waiting for already running chef-clients on #{node_name}.") if use_logger
```

Comment:
That's what I mentioned in my comment on the PR: it seems `@logger` is not available after the fork. But only on ruby 1.8.